packages/errors/src/errors.ts

```diff
@@ -620,22 +620,14 @@ export const AllCypressErrors = {
     return errTemplate`\
         We detected that the Chromium Renderer process just crashed.
 
-        This is the equivalent to seeing the 'sad face' when Chrome dies.
+        This can happen for a number of different reasons.
 
-        This can happen for a number of different reasons:
+        If you're running lots of tests on a memory intense application.
+          - Try increasing the CPU/memory on the machine you're running on.
+          - Try enabling ${fmt.highlight('experimentalMemoryManagement')} in your config file.
+          - Try lowering ${fmt.highlight('numTestsKeptInMemory')} in your config file during 'cypress open'.
```

Comment:
Do we want to actually flex the display of this line based on whether or not they are in open/run mode?

Reply:
We could if we want to spend more time on it and be smarter and we feel this message doesn't convey the intention properly.

Reply:
We decided to leave the new message as-is.